recipe/meta.yaml

```diff
@@ -26,6 +24,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cgo') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
Did you change the `conda-forge.yml` file to match (and then rerender the feedstock): https://conda-forge.org/docs/maintainer/knowledge_base.html?#using-centos-7

Reply:
(I did change conda-forge to match, yes)

Reply:
The rerender below forced the conda-forge edit back to cos6...

Reply:
> Good point! Rerendering now, had an error come up for the glibc stuff, so maybe a rerender will resolve that?

Yes

> The rerender below forced the conda-forge edit back to cos6...

You rerendered before changing the files to specify `cos7`

Reply:
@conda-forge-admin, please rerender

Reply:
YES! I think this is it! We are done here (we just need to clean up now)

Thanks a lot!!!

Reply:
@chrisburr should I clean up the manic commits in a new pr or should we just adopt this? I will also highlight to you soon some issues with the tests that we need to figure out. We ideally should not skip some jus to make it work, although it really shouldn't matter imo